src/components/Sidebar/components/Header/Header.js

```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+
+const baseStyles = ({ theme }) => css`
+  label: header;
+  display: flex;
+  align-self: flex-start;
```

Comment:
Ah! I see. Here things are aligning themselves inside the flex container. 👌 This is fine then. In people who don't have flexbox support on their device, should still get the default `block` behavior of the `div`s.